po/fr.po

```diff
@@ -3137,7 +3137,7 @@ msgstr "étendue spatiale"
 #: ../build/lib/darktable/plugins/introspection_colorreconstruction.c:68
 #: ../build/lib/darktable/plugins/introspection_colorreconstruction.c:131
 msgid "range extent"
-msgstr "étendue de clarté"
+msgstr "étendue de luminance"
```

Comment:
Note that the tooltip for this button is in English:

"how far to look for replacement colors in the luminance dimension"

Which seems to indicate that indeed luminance is better here.

Reply:
Well, the module operates in Lab, so it cannot be luminance, it's lightness. En français, luminosité ou clarté, je ne sais plus lequel traduit lightness.